test/debug_view.hpp

```diff
@@ -41,7 +41,7 @@ struct debug_input_view
       : debug_input_view{data, N}
     {}
     constexpr /*c++14*/
-    debug_input_view(debug_input_view &&that) noexcept
+    debug_input_view(debug_input_view const &that) noexcept
```

Comment:
This copy constructor doesn't meet the semantic requirements of `CopyConstructible`.

Reply:
Picky, picky.

Reply:
How about now?

Reply:
It looks right now.